aws-crt-kotlin/common/test/aws/sdk/kotlin/crt/auth/credentials/CredentialsProviderTest.kt

```diff
@@ -9,20 +9,23 @@ import aws.sdk.kotlin.crt.CrtTest
 import aws.sdk.kotlin.crt.io.ClientBootstrap
 import aws.sdk.kotlin.crt.io.EventLoopGroup
 import aws.sdk.kotlin.crt.io.HostResolver
-import aws.sdk.kotlin.crt.runSuspendTest
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Ignore
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 private val EXPECTED_CREDENTIALS = Credentials("access_key_id", "secret_access_key", "session_token")
 
 class CredentialsProviderTest : CrtTest() {
+    @Ignore // FIXME Enable when Kotlin/Native implementation is complete
```

Comment:
note: we haven't implemented Native credentials providers with bindings to CRT (and probably never will?) since we have **common** implementations in smithy-kotlin